### PR TITLE
Feat/rescrape events

### LINKED ISSUE #105: GCP - add cloud function and scheduler to re-scrape events
**Description**
Currently, we do not have an automated way to re-scrape the events we have. So, we will create another cloud functional and scheduler to periodically re-scrape the top 10 oldest city collections.

**TODO**
- [x] extract the current scraper logic into a non-cloud, helper function
- [x] create new cloud function that grabs the top 10 oldest city collection
- [ ] add collection to call this function once a day

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by lx1922**
### Issue
Closes #105 

### What is changed / added
- extract the current scraper logic into a non-cloud, helper function
- create new cloud function that grabs the top 10 oldest city collection

### CODE REVIEW

File: src/platform/functions/index.js [hunk 1]
```diff
@@ -194,3 +215,29 @@ exports.deleteOutdatedUserEvents = functions
     functions.logger.info('Deleting Successful', deletedEvents);
     res.send(deletedEvents);
   });
+
+exports.periodicRescraper = functions
+  .runWith({
+    timeoutSeconds: 60,
+    memory: '1GB',
```
Comment (TheFatPanda97) [inline src/platform/functions/index.js:222]:
```suggestion
    timeoutSeconds: 120,
    memory: '2GB',
```
let's bump this up, cloud functions sometimes fail with lower ram and timeout

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
looks good, just need to merge master

File: src/platform/functions/index.js (hunk 1 above)
Comment (lx1922) [reply src/platform/functions/index.js:222]: cool! done!

**Review by TheFatPanda97 — APPROVED** [review]
LGTM

**Pull request merged**